services/filemanager/azureBlobStoragemanager.go

```diff
@@ -131,6 +135,8 @@ func (manager *AzureBlobStorageManager) ListFilesWithPrefix(prefix string, maxIt
 	}
 
 	// List the blobs in the container
+	ctx := context.Background()
```

Comment:
Let's keep this on the top, a reminder that this method should accept a context at some point